Obsidian.API/_Interfaces/IServer.cs

```diff
@@ -22,6 +22,7 @@ public interface IServer
     public void BroadcastMessage(string message, MessageType type = MessageType.Chat);
     public void BroadcastMessage(ChatMessage message, MessageType type = MessageType.Chat);
     public IPlayer? GetPlayer(string username);
+    public IPlayer? GetPlayerIgnoreCase(string username);
```

Comment:
I feel like this should get removed completely as username checks should be case-sensitive by default

Reply:
I'm working on fixing it.